docs/reference/settings.rst

```diff
@@ -41,6 +41,24 @@ When ``WAGTAIL_APPEND_SLASH`` is ``False``, requests to Wagtail pages will be se
 
 .. _this Google Search Central Blog post: https://developers.google.com/search/blog/2010/04/to-slash-or-not-to-slash
 
+ADMIN BASE URL
+==============
+
+``WAGTAILADMIN_BASE_URL``
+--------------------------
+
+.. code-block:: python
+
+    WAGTAILADMIN_BASE_URL = 'http://example.com/'
+
+This is the base URL used by the Wagtail admin site. It is typically used for generating URLs to include in notification emails.
+
+If this setting is not present, Wagtail will try to fall back to ``request.site.root_url`` or to the requests host name.
+
+.. versionchanged:: 2.15
```

Comment:
sorry the 2.15 was just an example, the next release is 3.0
```suggestion
.. versionchanged:: 3.0
```

Reply:
oops sorry didn't realize